alacritty/src/display/window.rs

```diff
@@ -495,6 +495,17 @@ impl Window {
         self.windowed_context.resize(size);
     }
 
+    pub fn make_not_current(&mut self) {
+        if self.windowed_context.is_current() {
+            // SAFETY due to use of multiple windows we always ensure that the context is current
+            // before rendering, so treating it as current here to avoid extra complexity from
+            // addign extra enum.
```

Comment:
This doesn't prove safety at all? If this function is called outside of rendering, the window will not be current.

Reply:
> I think the comment is a bit misleading.

That's exactly my problem, the comment is misleading.